### ISSUE TITLE
App uninstall itself 

### ISSUE THREAD

**Issue by spislucka** [AdyRock/com.onvif#49]
Hi im having a problem. The app uninstall itself. Its happen twice now.  I usually notis the problem when the red ! mark appears on the cameras. After installing the app the cameras just stats working with no input from my side.

**Comment by AdyRock** [comment]
I don't even know how that is possible. 
Can you send a Diagnostics report for the app so I can ask Athom?

**Comment by spislucka** [comment]
Sure. f18aa777-3767-45cf-9aa8-050112de29815 apr. 2023 kl. 20:11 skrev Adrian Rockall ***@***.***>:﻿
I don't even know how that is possible.
Can you send a Diagnostics report for the app so I can ask Athom?

**Comment by AdyRock** [comment]
Can you try the test version by adding test to the end of the URL for the app in the store.

**Comment by spislucka** [comment]
1088e766-f628-419a-9e20-79effd58226d5 apr. 2023 kl. 21:34 skrev Adrian Rockall ***@***.***>:﻿
Can you try the test version by adding test to the end of the URL for the app in the store.

**Comment by AdyRock** [comment]
It seems the app can't connect to your car park camera. Is the camera turned on and still on the same IP address?

**Comment by spislucka** [comment]
Yes I noticed that. I believe it’s just the wrong password. That’s not the issue though, but maybe connected?5 apr. 2023 kl. 22:17 skrev Adrian Rockall ***@***.***>:﻿
It seems the app can't connect to your car park camera. Is the camera turned on and still on the same IP address?

**Comment by AdyRock** [comment]
The error is EHOSTUNREACH which normally means the device is not responding at all, so doesn't seem like a password issue. 
I'm not sure if it is related but I have been getting a lot of crash reports due to that type of error. 
The last 4 or 5 updates have all been trying to resolve that. I have tried reproducing the error but for me it is all handled correctly and I can't get it to crash so the updates have been a stab-in-the-dark.

**Comment by spislucka** [comment]
It has not disappeared since last night so that's good :)

Den ons 5 apr. 2023 kl 23:34 skrev Adrian Rockall ***@***.***
